### PR TITLE
Add new Card component

### PR DESCRIPTION
**Pull request by williamkolmacka**
This Pull Request meets the following criteria:

* [x] Tests have been added/adjusted for my new feature
* [x] New Components are registered in index.js of my project

Add a new Card component.

Closes: #119 and #120

### CODE REVIEW

**Comment by jaroslav-kubicek** [comment]
I might be missing something, but how I'm supposed to use `CardSection` in my code?

I would expect something like
```
<Card>
  <Card.Section />
</Card>
```

but section seems to be not exported

File: src/Card/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  position: relative;
+  width: 100%;
+  padding: ${props => props.theme.spaceLarge};
+  padding-right: ${props => (props.closable ? props.theme.spaceXXLarge : props.theme.spaceLarge)};
+  box-sizing: border-box;
+`;
+
+const StyledCardBody = styled.div`
+  width: 100%;
+  padding: ${props =>
```
Comment (jaroslav-kubicek) [inline src/Card/index.js]: this function is not too readable, would be nicer to define it as normal named function and avoid nested ternary operator

Comment (williamkolmacka) [reply src/Card/index.js]: Fixed.

File: src/Card/Card.stories.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <ThemeProvider>
+                  <Card
+                    icon={<Icons.Airplane color="attention" />}
+                    title={title}
+                    description={description}
+                    sectioned
```
Comment (wEprous) [inline src/Card/Card.stories.js]: Can we get rid of this `sectioned` prop?
We can check if children contains `Section` component - something like this:
```js
let sectioned = false;
children.map(child => {
  if (child.type === Section) {
    sectioned = true;
  }
  return sectioned;
});
```

Comment (wEprous) [reply src/Card/Card.stories.js]: Let me know @williamkolmacka

File: src/Card/CardSection.js.flow [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// @flow
+import defaultTokens from "@kiwicom/orbit-design-tokens"
+
+export type Props = {|
+  children: React$Node,
+  theme?: typeof defaultTokens
+|}
+
+export type styledCardSection = {
```
Comment (wEprous) [inline src/Card/CardSection.js.flow]: Is it necessary to export these types?

File: src/index.js [hunk 4]
```diff
@@ -18,6 +18,13 @@ export { default as Button } from "./Button";
 export { default as ButtonLink } from "./ButtonLink";
 export { default as CarrierLogo } from "./CarrierLogo";
 export { default as Checkbox } from "./Checkbox";
+
+// Card
+export { default as Card } from "./Card";
+export { default as CardHeader } from "./Card/CardHeaderWithTheme";
```
Comment (wEprous) [inline src/index.js]: As we talked, move subcomponents into separated folders, eg. `Card/CardHeader/`

**Pull request merged**